### ISSUE TITLE
Does APIARY allowed Multiple Host in same project

### ISSUE THREAD

**Issue by teja5429** [apiaryio/api-blueprint#422]
Hi,

I am trying to add two hosts under the same project. 
Host 1 : https://abc.com/
Host 2 : https://efg.com/

Currently I am able to work with one host by using "HOST: https://abc.com" , but I need two hosts configured in the same path. Is there any way to define two hosts in same project?

Thanks in Advance. Please help me out of this.

Regards,
Teja

**Comment by alflennik** [comment]
Also curious about this. We are using multiple test environments and being able to switch between them to find differences is critical.

**Comment by char0n** [comment]
We're not currently supporting this, but we're working on solutions that will make this possible in future.

**Comment by tejaschandra84** [comment]
Any update on this?

**Comment by ZilvinasKucinskas** [comment]
Are there any solutions to this problem already?

**Comment by Almad** [comment]
Not a public one; we do brainstorm about how to solve this problem and we are not sure multiple HOSTs are the correct solution, as opposed to decoupling design and deployment concerns.

**Comment by ZilvinasKucinskas** [comment]
@Almad I will describe my use case to give some feedback in regards to this problem.

Our Apiary use case is purely embedding pretty documentation to our developer dashboard. We are not planning on using any additional functionality, plugins or extensions right now. We use multiple subdomains for different parts of our system (micro services), for example, authentication, billing, statistics, resource retrieval, etc.. As you can see each subdomain is responsible for different domain.

What seems the most attractive about Apiary is the console functionality. You can describe endpoints and end-users can experiment with them through the console. It also adds an option to change languages and copy example code in different languages.

However, without an option to use multiple subdomains, we are left only with a few options:
* Use different document per micro-service. But then the documentation is scattered across the place. It's hard to follow. And linking between them is not trivial.
* Do not use the reference at all. Hide the console while embedding the documentation. The drawback is of course not showing console. Then, I am left thinking the next bullet point:
* Building API documentation design ourselves without using 3rd party.

None of the options seems great. Any recommendation to our case?

**Comment by Almad** [comment]
There will be a solution for it, but unfortunately I can't talk about it and give timelines because SEC and lawyers :( 

Would foreseeable future give sufficient hope?

**Comment by knyttl** [comment]
is there any progress?

**Comment by opichals** [comment]
Support for `host` object has been [added in API elements](https://github.com/apiaryio/api-elements/pull/62) primarily to support OAS3 Server Object mapping. Stay tuned for https://help.apiary.io/tools/interactive-documentation-v4/ renderer updates.

I haven't seen any progress for support in API Blueprint though.

**Comment by BackendDev-Auction** [comment]
Any update on these features.

**Comment by hdamico** [comment]
Any update on this?

**Comment by Famin42** [comment]
+1

**Comment by sssandyad** [comment]
+1

**Comment by oscarguinane** [comment]
+1

**Comment by shakir-fattani** [comment]
+1

**Comment by Nickyzj628** [comment]
+1
